_posts/2023-07-21-yanina-community.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title:  Let's Talk Community with Yanina Belline Saibene
+description: Discussion with Yanina on what makes a community
+date:   2023-07-21 10:01:35 +0300
+image:  '/images/blogs-community/yanina_comm_sq.png'
+tags:   [opensource, community]
+---
+
+Author: [Reshama Shaikh](https://reshamas.github.io)  
+
+We speak with [Yanina Belline Saibene](https://www.linkedin.com/in/yabellini/) about her exerience in communities.
+
+### About Yanina Belline Saibene
+
+[add bio here and include the communities you are a member of, including links to them?]
```

Comment:
```suggestion
[Yanina Bellini Saibene](https://yabellini.netlify.app/about/) is the [rOpenSci](https://ropensci.org/) Community Manager and [R-Ladies](https://rladies.org/) project lead + Leadership Team. She teaches coding skill since 1993. She co-founded MetaDocencia and [LatinR](https://latin-r.com/) and is a member of [The Carpentries](https://carpentries.org/index.html) Executive Council. She is also a member of the [R Consortium Infrastructure Steering Committee](https://www.r-consortium.org/about/governance), [MiR](https://mircommunity.com/), [Open Heroines](https://openheroines.org/) and [Sociedad Argentina de Informática (SADIO)](https://www.sadio.org.ar/). She is Topic Editor at [Journal of Open Source Education](https://jose.theoj.org/) and Scientific Committe [Electronic Journal of SADIO](https://jose.theoj.org/). She teach at [UNAB](https://www.unab.edu.ar/analisis-datos.html), [Autral](https://www.austral.edu.ar/ingenieria-posgrados/cuerpo-docente/mag-yanina-bellini-saibene/), [UCES](https://w.uces.edu.ar/wp-content/uploads/2022/08/Agroanalytics.pdf), and [UDE](https://online.fca-ude.edu.uy/course/index.php?categoryid=66) Universities. She lead the multilingual project at rOpenSci and is co-editor of the translation of [Teaching Tech Together](https://educarencomunidad.tech/). She volunteer to bring R4DS, The Carpentries lessons, R-Ladies material, and Posit Cheatsheets to Spanish-speaking audience. She also enjoys organizing event and [conferences](https://yabellini.netlify.app/conferences/) and [talk](https://yabellini.netlify.app/talk/) about her many projects.
```